app/models/chef_ingredient.rb

```diff
@@ -0,0 +1,7 @@
+class ChefIngredient < ApplicationRecord
+    
+    def the_ingredients(chef_id)
```

Comment:
I see you were trying to create a relationship directly between Chef and Ingredient. You were on the right track! But you can set up this direct connection by adding the additional association from the existing chef model